modules/core/src/main/scala/scala/build/errors/CantDownloadAmmoniteError.scala

```diff
@@ -0,0 +1,18 @@
+package scala.build.errors
+
+import coursier.error.ResolutionError
+
+import scala.build.Position
+
+case class CantDownloadAmmoniteError(
```

Comment:
Exceptions shouldn't be case classes (because they hold a stack trace of the location where they've been instantiated, so that calling `copy()` on an instance returns an exception with a different stack trace - also, the `equals` method is wrong, making equal two exceptions with the same parameters but different stack traces)
```suggestion
final class CantDownloadAmmoniteError(
```

Reply:
fixed